packages/json-schema/test/models.test.ts

```diff
@@ -254,4 +254,41 @@ describe("emitting models", () => {
       $ref: "RecordInt32.json",
     });
   });
+
+  describe("default values", () => {
+    it("specify default value on enum property", async () => {
+      const res = await emitSchema(
+        `
+        model Foo {
+          optionalEnum?: MyEnum = MyEnum.a;
+        };
+        
+        enum MyEnum {
+          a: "a-value",
+          b,
+        }
+        `
+      );
+
+      deepStrictEqual(res["Foo.json"].properties.optionalEnum, {
+        $ref: "MyEnum.json",
+        default: "a-value",
+      });
+    });
+
+    it("specify default value on string property", async () => {
```

Comment:
What about other properties? You've got cases for number, tuple, boolean and instrinsic but test cases only for enum and string.